src/app/api/session/active/complete/route.ts

```
@@ -1,24 +1,11 @@
-import { NextRequest, NextResponse } from 'next/server';
-import { auth } from '@clerk/nextjs/server';
+import { NextRequest } from 'next/server';
```

ESLint

[error] 'NextRequest' is defined but never used.

(@typescript-eslint/no-unused-vars)